dt-impact-tracer/src/edu/washington/cs/dt/impact/Main/EnhancedResultsFigureGenerator.java

```diff
@@ -77,27 +81,35 @@ private static String generate18(String projectName, double[] percentages, doubl
              * double percent = Math.max(enhancedParaSpeedup, Math.max(unenhancedPara, diffBetweenEnhancedUnenhanced));
              */
             double percent;
+            // i represents unenhanced, i + 1 represents enhanced
             if (!nonZeroNumOfDTS[i]) {
-                percent = (percentages[i + 1] - percentages[i]) / orig_time_value * 100;
+                // unenhanced - enhanced
```

Comment:
This is not right. It should be enhanced - unenhanced.
There is no reason why we would compare unenhanced with original. Our technique is the enhanced one.